repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java

```diff
@@ -393,7 +398,7 @@ private void initialize(AtlasGraph graph) throws RepositoryException, IndexExcep
             createCommonVertexIndex(management, TASK_GUID, UniqueKind.GLOBAL_UNIQUE, String.class, SINGLE, true, false);
             createCommonVertexIndex(management, TASK_TYPE_PROPERTY_KEY, UniqueKind.NONE, String.class, SINGLE, true, false);
             createCommonVertexIndex(management, TASK_CREATED_TIME, UniqueKind.NONE, Long.class, SINGLE, true, false);
-            createCommonVertexIndex(management, TASK_STATUS, UniqueKind.NONE, String.class, SINGLE, true, false);
+            createCommonVertexIndex(management, TASK_STATUS, UniqueKind.NONE, String.class, SINGLE, true, false, false, new HashMap<>(), KEYWORD_FIELD);
```

Comment:
Just for reminder this change will only work for new tenants only! Not for old tenants as the mapping in ES already been created.

Reply:
SO you need to run simple migration to add this field in ES mapping in all the existing tenants, we already did this. NOTE: This is just mapping update won't take much time.
Then if we want this for other existing vertex we need reindex the fields.

Reply:
That serves the purpose, atleast in playbook use-case we have nothing to do with past tasks that are already done. This [PR](https://github.com/atlanhq/marketplace-packages/pull/12604/files) should handle adding the mapping correct?